todo_app/item.py

```diff
@@ -1,6 +1,6 @@
 class Item:
 
-    def __init__(self, id, title):
+    def __init__(self, id, title, started):
         id = id
-        status = "Not started"
+        started = False
         title = title
```

Comment:
Note that now we're using this class, your app doesn't display anything correctly! It's highlighted that we've got a mistake here - we're not currently storing the fields on the class. I think this should be:
```
    def __init__(self, id, title, started):
        self.id = id
        self.started = False
        self.title = title
```